docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -81,6 +81,16 @@ An add-in that uses the unified manifest may not have more than 20 [add-in comma
    > [!IMPORTANT]
    > The value of this element must exactly match the name of an action that's mapped to a function in a JavaScript or TypeScript file with the [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member(1)) function. If you change it in the manifest, be sure to change it in the `actionId` parameter passed to `associate()` too.
 
+### Shorten string values as needed
+
+Review and change, as needed, the manifest values in light of the following effects of the conversion.
+
+- The first 30 characters of `<DisplayName>` will become the value of "name.short" in the unified manifest.
```

Comment:
I think we can use present tense here (and align to the style guide) without losing meaning, but let me know if you disagree. 
```suggestion
- The first 30 characters of `<DisplayName>` becomes the value of "name.short" in the unified manifest.
```